.idea/codeStyles/Project.xml

```diff
@@ -15,11 +15,11 @@
     <HTMLCodeStyleSettings>
       <option name="HTML_ENFORCE_QUOTES" value="true" />
     </HTMLCodeStyleSettings>
-    <JSCodeStyleSettings version="0">
+    <JSCodeStyleSettings>
```

Comment:
These settings keep changing based on the IntelliJ version that we use.
Not sure whether we should commit these changes or ignore.

\cc: @ketan